metrics/nop_reporter.go

```diff
@@ -99,3 +103,30 @@ type nopCachedTimer struct {
 
 func (nopCachedTimer) ReportTimer(interval time.Duration) {
 }
+
+// NopCachedHistogram is an implementation of tally.CachedHistogram
```

Comment:
Should we do this everywhere in this file? I'd prefer that, but not sure, if it will confuse reviewers.